src/main/java/seedu/address/logic/parser/CliSyntax.java

```diff
@@ -16,4 +16,6 @@ public class CliSyntax {
     public static final Prefix PREFIX_PROPERTIES = new Prefix("-pr");
     public static final Prefix PREFIX_PRICE = new Prefix("-price"); // temporary, should refactor all prefixes
     public static final Prefix PREFIX_DESCRIPTION = new Prefix("-d");
+    public static final Prefix PREFIX_PROPERTY = new Prefix("-prop");
+    public static final Prefix PREFIX_BUYER = new Prefix("-buyer");
```

Comment:
Where are these prefixes used? Do our commands actually need this?

Reply:
No, I thought about whether to make the command behave such that list -prop list the property list and list -buyer list the buyer list. But I decided to make two commands in the end. Will delete!